docs: update docs about `gatsby-plugin-eufemia-theme-handler` (#2740)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/brand/development/import-styles.mdx

  </Theme>,
)
```

## Runtime theme swap

> How ever, the above solution will not work for changing the theme in runtime.

Reviewer: ```suggestion
However, the above solution will not work for changing the theme in runtime.
```

> Changing a theme in runtime, without pre-loading all CSS styles and fonts together, requires a more sufisitcated solution.

Reviewer: ```suggestion
Changing theme during runtime, without pre-loading all CSS styles and fonts together, requires a more sophisticated solution.
```

> When using Gatsby, you can use this plugin [gatsby-plugin-eufemia-theme-handler](https://github.com/dnbexperience/gatsby-plugin-eufemia-theme-handler).

Reviewer: ```suggestion
When using Gatsby, you can use this plugin [gatsby-plugin-eufemia-theme-handler](https://github.com/dnbexperience/gatsby-plugin-eufemia-theme-handler) to change theme during runtime.
```
